azure-data-lake/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,78 @@
+= Azure Data Lake Storage Connector - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Azure Data Lake Storage Gen 2 (Azure Data Lake Storage Connector) provides access to standard Azure Data Lake Storage Gen 2 operations using Anypoint Platform. 
+
+Azure Data Lake Storage Gen2 is a scalable data storage service built by Microsoft Azure and designed for big data analytics. Azure Data Lake Storage Gen2 is built on top of Azure Blob Storage and provides the data organization and security semantics of Azure Data Lake Gen1 along with the cost and reliability benefits of Azure Blob Storage.
+
+For information about compatibility and fixed issues, see Azure Data Lake Storage Connector Release Notes. 
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* Azure Data Lake Storage Gen2 API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* How to create a Mule app using Anypoint Studio
+
+Before creating an app, you must have access to:
+
+* The Azure Data Lake Storage target resource
+* Anypoint Platform
+* Anypoint Studio version 7.1 or later
+
+
+== Common Use Cases for the Connector
+
+Common use cases for Azure Data Lake Storage Connector include the following: 
+
+* Create, read, update, and delete files in an existing Azure Data Lake Storage file system.
+* Create, rename, and delete directories from the Azure Data Lake Storage file system.
+* Rename existing files. 
+* Append the contents of a file and flush the file's contents.
+* Obtain properties and status for a file or directory.
+* Obtain the ACL (access control list) of a file or directory.
+* Get a list of all the available paths in the file system.
+
+For examples of some of these common use cases, see xref:azure-data-lake-connector-examples.adoc[Examples].
+
+== Audience
+
+* New user
++
+To create your Mule app, see:
+
+* xref:azure-data-lake-connector-studio.adoc[Studio Configuration] 
+* xref:azure-data-lake-connector-examples.adoc[Examples], with use case information for the connector
++
+* Advanced user
++
+* Read the following: 
+
+* xref:azure-data-lake-connector-xml-maven.adoc[XML and Maven Support]
+* xref:azure-data-lake-connector-examples.adoc[Examples], with use case information for the connector
+
+== Connection Types
+
+Azure Data Lake Storage Connector connections use the following authentication types:
+
+* Azure Active Directory OAuth 2.0 Connection Provider +
```

Comment:
I'd change this name to just "Azure Active Directory OAuth 2.0" without the connection provider, in all the parts of the document that mentions it